MapboxNavigation/RouteVoiceController.swift

```diff
@@ -159,14 +161,24 @@ open class RouteVoiceController: NSObject, AVSpeechSynthesizerDelegate {
     }
     
     func duckAudio() throws {
-        let categoryOptions: AVAudioSessionCategoryOptions = [.duckOthers, .interruptSpokenAudioAndMixWithOthers]
-        try AVAudioSession.sharedInstance().setMode(AVAudioSessionModeSpokenAudio)
-        try AVAudioSession.sharedInstance().setCategory(AVAudioSessionCategoryPlayback, with: categoryOptions)
+        if #available(iOS 12.0, *) {
```

Comment:
Looks like the Xcode 9 build bots are failing, since this is the first availability check for iOS 12 that we’ve added. It’s probably time to jettison the Xcode 9 build bots in favor of Xcode 10, given that we aren’t far from the next version of Xcode.

Reply:
Agree.